### PR TITLE
Add RSPEC to project

### PR DESCRIPTION
**Pull request by hieunn-3093**
## Related Tickets
[#TicketID](https://edu-redmine.sun-asterisk.vn/issues/47771)
## WHAT (optional)
Thêm Rspec vào project

- cài gem
- gem "simplecov-rcov"
- gem "simplecov"
- gem "shoulda-matchers", "~> 5.0"
-  gem "database_cleaner"
-  gem "factory_bot_rails"

## Evidence (Screenshot or Video)
Screenshot from 2022-03-30 10-10-38

### CODE REVIEW

**Comment by hieunn-3093** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -22,6 +22,8 @@ gem "puma", "~> 4.1"
 gem "rails", "~> 6.0.4", ">= 6.0.4.6"
 gem "rails-i18n"
 gem "sass-rails", ">= 6"
+gem "simplecov"
+gem "simplecov-rcov"
```
Comment (ghost) [inline Gemfile:26]: Thêm database cleaner nữa em nhé

**Review by ghost — COMMENTED** [review]
cmt

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**